samples/http-json-cloudevents/README.md

```diff
@@ -0,0 +1,20 @@
+## Quickstart
```

Comment:
Make titles unique so you don't need to read the filename to understand what this is about.

Reply:
Renamed Quickstart to Sending cloudevents with image payloads to a server